<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyAutoMaskingPIFormClass</class>
 <widget class="GUIFormBase" name="PropertyAutoMaskingPIFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyAutoMaskingPIForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>381</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="ButtonLibNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear new</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyAutoMaskingPI.qrc">
      <normaloff>:/Qtres/ClearNew.png</normaloff>:/Qtres/ClearNew.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Library Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBDefaultMask">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>デフォルト</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLibSave">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyAutoMaskingPI.qrc">
      <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLibDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyAutoMaskingPI.qrc">
      <normaloff>:/Qtres/Delete.png</normaloff>:/Qtres/Delete.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>収縮ドット（外形の輪郭部分の除外のため）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditEnSmallDot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLibSaveNew">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save New</string>
    </property>
    <property name="icon">
     <iconset resource="PropertyAutoMaskingPI.qrc">
      <normaloff>:/Qtres/SaveNew.png</normaloff>:/Qtres/SaveNew.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>抽出する輝度差のパーセント（％）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>無効化させたい最小スリット・穴の面積（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditColDiffRate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinSlitDot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibID">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>膨張ドット（小径穴を検査エリアに含めるため）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditEnLargeDot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>無効化させたい最小スリット・穴の長さ（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinSlitSize">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinSlitWidth">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>無効化させたい最小スリットの幅（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMaxLakeSize">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>孤島のある湖の最大長さ（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinClusterSize">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>有効エリアの最小長さ（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>有効エリアの最小面積（ドット）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinClusterArea">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>影部分（基板無しより暗い部分）の抽出</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBPickupDarker">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>ノイズサイズ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditNoizeSize">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>外形の輪郭部分がPAD抽出色であっても収縮</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkEnLargeDot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QToolButton" name="toolButtonIneffectiveMask">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>手動無効マスク</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAutoMask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>自動生成マスク</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonEffectiveMask">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>手動有効マスク</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetMaskingList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Page</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Effective</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Area</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="NoRole">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>表示</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonCompositeMask">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>結果合成マスク</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerateLib">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>選択ライブラリで生成</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyAutoMaskingPI.qrc">
     <normaloff>:/Qtres/Add.png</normaloff>:/Qtres/Add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerateDefault">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最初のデフォルトで生成</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyAutoMaskingPI.qrc">
     <normaloff>:/Qtres/Add.png</normaloff>:/Qtres/Add.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyAutoMaskingPI.qrc"/>
 </resources>
 <connections/>
</ui>
